docker/ebpf-probe-builder/build_bpf_probe.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+mkdir -p ${HOME}/.sysdig
+
+docker build -t ebpf-probe-builder:latest --pull .
+docker run --rm -it -v ${HOME}/.sysdig:/out -v /opt/draios/src/draios-agent-0.1.1dev:/driver -v /lib/modules/$(uname -r)/build:/kernel -v /:/host:ro -e BPF_PROBE_FILENAME=bpf_probe.o ebpf-probe-builder:latest
```

Comment:
In my opinion, a generic way would be to just take `/opt/draios/src/draios-agent-0.1.1dev` and `/lib/modules/$(uname -r)/build` as input parameters of the script (e.g. $1, $2) instead of leaving them hardcoded. For your tests, you'll call it using these two arguments, whereas for the probe builder we'll call it using the kernel path we're processing at the moment, and the sysdig driver's working directory right after the cmake bootstrap.

Reply:
Also, do we still need the mount against `/`?